debugging/debugging.md

```diff
@@ -45,7 +45,7 @@
 - If the red squiggly underline is under something that's clearly correct, like `color: red;` in a CSS file, then first check if it really is correct.
 - The phrase "pattern matching" and why it's important: use other code snippets to remind yourself of correct syntax.
 - When you make a change on a file, and then visually when you refresh your page you don't see any changes, run through a quick checklist:
-    - Did you save? this kills me every time (see: add autosave to VS Code lesson if it's not already there).
+    - Did you save? this kills me every time ([add autosave to VS Code lesson if it's not already there](https://code.visualstudio.com/docs/editor/codebasics#_save-auto-save)).
```

Comment:
so, these are the issue directions, not literally what you should put in the lesson.  Can you think about how to reword this for learners?